easybuild/easyblocks/m/metis.py

```diff
@@ -47,14 +48,17 @@ class EB_METIS(ConfigureMake):
     def __init__(self, *args, **kwargs):
         """Define custom class variables for METIS."""
         super(EB_METIS, self).__init__(*args, **kwargs)
-
         self.lib_exts = []
 
     def configure_step(self, *args, **kwargs):
         """Configure build using 'make config' (only for recent versions (>= v5))."""
 
         if LooseVersion(self.version) >= LooseVersion("5"):
 
+            if build_option('rpath'):
+                # patch Makefile to tell CMake not to wipe the RPATHs we inject...
+                apply_regex_substitutions('Makefile', [(r'^(CONFIG_FLAGS\s*=\s*)', r'\1-DCMAKE_SKIP_RPATH=ON ')])
```

Comment:
Should be there already, but doesn't hurt to make sure, you're right, I'll add it.